addons/osfstorage/tests/test_views.py

```diff
@@ -36,8 +39,8 @@
 
 def create_record_with_version(path, node_settings, **kwargs):
```

Comment:
This function wasn't being used at all before, so I just changed it slightly and used it because it looked relevant.